### DISCUSSION TITLE
BPS plugin E+: about the building envelope parameters

### DISCUSSION THREAD

**Discussion by Testerbps** [BIM2SIM/bim2sim#538] [Q&A] [answered]
hi development team,
now for change the building parameters part, how could user change the U-value of the walls.
that could be directly set or written in Bim2sim or be just set in E+.
![image](https://github.com/BIM2SIM/bim2sim/assets/133635055/0f00afe4-4266-42da-af5d-cee03c518445)

Regards
tester

**Comment by veronikarichter** [comment, marked as answer]
Hi tester, 
I would recommend you to set these parameters in the resulting IDF (either manually or using python packages like eppy).
